numbat-cli/src/main.rs

```diff
@@ -24,12 +24,27 @@ use rustyline::{
     Validator,
 };
 use rustyline::{EventHandler, Highlighter, KeyCode, KeyEvent, Modifiers};
+use serde::Deserialize;
+use toml;
 
 use std::io::IsTerminal;
 use std::path::PathBuf;
 use std::sync::{Arc, Mutex};
 use std::{fs, thread};
 
+#[derive(Deserialize)]
+struct Config {
+    #[serde(
+        default = "default_prefetch_exchange_rates",
+        rename = "prefetch-exchange-rates"
```

Comment:
I prefer kebab case, but I'm also fine with using snake case if you want.

Reply:
Sounds good. You can do this for all fields using `#[serde(rename_all = "kebab-case")]`